### ISSUE TITLE
Change branch before delete

### ISSUE THREAD

**Issue by RuZniki** [neclimdul/drupal_reroll#2]
As I remember it's impossible to delete current branch, so you need to change branch and then delete.

**Comment by neclimdul** [comment]
you are correct. I'm not sure its going to be safe to make assumptions about what branches the user has though. Solutions might be:

1. error out
1. pick the first branch listed by 'git branch'
